pom.xml

```diff
@@ -161,6 +161,9 @@
 
         <version.commons-digester>1.8.1</version.commons-digester>
         <maven.surefire.plugin.version>2.6</maven.surefire.plugin.version>
+
+        <version.org.springframework>3.2.12.RELEASE</version.org.springframework>
```

Comment:
The property is there https://github.com/jboss-integration/jboss-integration-platform-bom/blob/6.0.0.Final/pom.xml#L343 so something else must be going on. Let me try this locally.